pyproject.toml

```diff
@@ -64,6 +64,7 @@ line_length = 160
 
 [project.optional-dependencies]
 pg_notify = ["psycopg[binary]>=3.2.4"]
+metrics = ["uvicorn[standard]", "prometheus-client"]
```

Comment:
should that really be optional?

Reply:
A metrics server via prometheus-client is kind of a boutique thing.

Maybe I'm being a little too die-hard about this, but imagine that someone specifies a metrics class in the config that was something entirely different, using an entirely different stack. As long as that class conformed to the protocol, it could run and serve something entirely different on some port.